Makefile

```diff
@@ -27,3 +27,11 @@ test:
 .PHONY: integration-test
 integration-test:
 	./gradlew -Pojc.integration.tests=true test
+
+.PHONY: maven-publish
+maven-publish:
```

Comment:
Not specific for this PR, but why do you use make? :)

Reply:
it is certainly not "more trouble", just IMO, if "users should be using `./gradlew -PmagicFlag=magic -x check -x randomTask assemble specialTest ` instead of `./gradlew build`", then we should fix gradle build, not introduce another layer on top of it :)

Reply:
We're hopefully not already at diminishing returns, but I appreciate your perspective: https://github.com/open-telemetry/opentelemetry-java-contrib/issues/11.